src/finiteVolume/processes/les.cpp

```diff
@@ -68,6 +68,8 @@ PetscErrorCode ablate::finiteVolume::processes::LES::LesMomentumFlux(PetscInt di
     PetscReal turbulence;
 
     turbulence = aux[aOff[TKE_FIELD]];
+    // Bound TKE field if it is gone unbounded.
+    turbulence = fmax(0, turbulence);
```

Comment:
For petsc values you will want to user PetscMax instead of max.

Reply:
I see. Thank!